docs/examples/network/complete-network-example.py

```diff
@@ -58,7 +58,7 @@ def createNetwork(dataSource):
   network = Network()
 
   # Add a sensor region.
-  network.addRegion("sensor", "py.RecordSensor", '{}')
+  network.addRegion("sensor", "py.record_sensor", '{}')
```

Comment:
@rhyolight, this change and othes like it are the culprit. You should leave them alone. "py.RecordSensor" refers to the class name, not the module name. The registration logic then locates the module name that matches the class name.